ontopy/utils.py

```diff
@@ -835,3 +835,34 @@ def recur(o):
                     )
 
     return layout
+
+
+def copy_annotation(onto, src, dst):
+    """In all classes and properties in `onto`, copy annotation `src` to `dst`.
+
+    The `src` and `dst` can either be provided as a label string or a full IRI.
+    """
+    if onto.world[src]:
+        src = onto.world[src]
```

Comment:
Why check in world?

Reply:
Because we want to support src of the form `http://www.w3.org/2004/02/skos/core#prefLabel`